readme.md

```diff
@@ -37,3 +37,5 @@ add additional user config options
 consistent ordering of user states
 
 cleanup documentation in README file
+
+update tests for new email sending flow
```

Comment:
this looks good so far

Reply:
what do you think about the second test?